### DISCUSSION TITLE
Cara Menyimpan File .rmd ke word/pdf

### DISCUSSION THREAD

**Discussion by sunashadi** [teamalgoritma/community#279] [Q&A] [answered]
Minta tolong cara menyimpan file.rmd ke file word/pdf. Beberapa kali saya coba selalu gagal. Terima kasih.

**Comment by anugrah18003** [comment, marked as answer]
Dear @sunashadi 

Silahkan untuk mencoba langkah berikut:

1. Install Tex distribution menggunakan **tinytex::install_tinytex()**
2. Menambah beberapa pengaturan pada default YAML menjadi seperti:
![image](https://user-images.githubusercontent.com/70314445/186822085-b89d040c-479a-479e-8585-19a7597efbdb.png)
3. Silahkan untuk dicoba di knit. 
Note: Pastikan didalam .Rmd tidak ada yang error.

Reference: https://stackoverflow.com/questions/58427254/failed-to-compile-test-tex-see-https-yihui-name-tinytex-r-debugging-for-debu
